spec/commands/admin/update_code_group_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module AnonymousCodes
+    module Admin
+      describe UpdateCodeGroup do
+        let(:current_organization) { create(:organization) }
+        let(:current_user) { create(:user, :confirmed, :admin, organization: current_organization) }
+        let(:form_params) do
+          {
+            title: "Sample Code Group",
```

Comment:
```suggestion
            title_en: "Sample Code Group",
```
As this is a translatable field, the form is probably expecting this format (along with the other languages `title_ca`, `title_es`). Check other examples in Decidim

Reply:
Done